.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I wanna say that .vscode should be in the .gitignore, currently, it just ignores .vs, if you're able, add .vscode to the .gitignore

Reply:
As Noah said, this should be added to .gitignore.